examples/train_policy.py

```diff
@@ -61,7 +61,7 @@
     parser.add_argument(
         '--vector-env-nums',
         type=int,
-        default=2,
+        default=1,
```

Comment:
The default value of ``vector-env-nums`` needs to be 1 because the early-terminated algorithms does not support vector environments. The original default value 2 will trigger error, so we set 1 instead.